### PR TITLE
Update docs outlining basic alembic usage

### PR DESCRIPTION
**Pull request by acrellin**
Closes https://github.com/skyportal/skyportal/issues/1222

### CODE REVIEW

File: doc/migrations.md [hunk 1]
```diff
@@ -5,26 +5,32 @@
 If you are planning to use database migrations, you need to let
 Alembic know the current state of the database.
 
-Presuming you've just started off by running `make load_demo_data`,
+Presuming you've just started off by running `make load_demo_data`
+on the latest master branch commit (do not check out your database schema changes yet),
```
Comment (stefanv) [inline doc/migrations.md:9]: What does "do not check out your database schema changes yet" mean?

Perhaps: "(this should happen on a vanilla master branch, i.e. *without* any of your changes to the database schema)"

File: doc/migrations.md [hunk 2]
```diff
@@ -5,26 +5,32 @@
...
 ```
 PYTHONPATH=. alembic -x config=config.yaml stamp head
 ```
 
-## Upgrade to latest
+## Generate migration script
 
-Subsequently, when the database schema changes, run the following to
-upgrade:
+To generate a migration script, after having stamped the latest master commit (above),
```
Comment (stefanv) [inline doc/migrations.md:18]:
```suggestion
To generate a migration script, after having stamped the latest master commit (see above),
```

File: doc/migrations.md [hunk 3]
```diff
@@ -5,26 +5,32 @@
...
+changes, and run the following:
 
 ```
-PYTHONPATH=. alembic -x config=config.yaml upgrade head
+PYTHONPATH=. alembic -x config=config.yaml revision --autogenerate -m "Revision description"
 ```
 
-## Generate migration
+Review the resulting migration file under `alembic/versions` at hand of the [documentation](https://alembic.sqlalchemy.org/en/latest/autogenerate.html).
+
```
Comment (stefanv) [inline doc/migrations.md:27]:
```suggestion
```

File: doc/migrations.md [hunk 4]
```diff
@@ -5,26 +5,32 @@
...
+
+## Applying migration scripts
+
+Subsequently, after ensuring that the app is stopped, check out the branch with
+pending migration scripts, and run the following to upgrade:
```
Comment (stefanv) [inline doc/migrations.md:32]: Should proably add a note about migration manager, noting that this happens automatically upon firing up the app, and that the file `log/migration.log` shows more details.

**Comment by acrellin** [comment]
@stefanv thanks for the review -- all suggestions applied!

**Review by stefanv — APPROVED** [review]

**Review by dmitryduev — APPROVED** [review]
👍🏻

**Pull request merged**